Gemfile.lock

```diff
@@ -9,6 +9,59 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
+    actioncable (6.1.3.2)
```

Comment:
What do you think about fixing the version in the Gemfile? also since our target applications are rails >= 7.0 shall we update the references here?